engine/src/stack.h

```diff
@@ -1083,7 +1084,9 @@ class MCStack : public MCObject
 	void SetSystemWindow(MCExecContext& ctxt, bool setting);
 	void GetMetal(MCExecContext& ctxt, bool& r_setting);
 	void SetMetal(MCExecContext& ctxt, bool setting);
+	using MCObject::GetShadow;
```

Comment:
Option 3 is preferable - it makes it completely obvious it is a different function without fiddling with scopes.